test11.js

```diff
@@ -0,0 +1,10 @@
+var foo = '' ;
```

Comment:
Codacy has a fix for the issue: [Replace `''·` with `""`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5795)
```suggestion
var foo = "";
```